src/vm/i386/excepx86.cpp

```diff
@@ -3731,4 +3752,10 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifdef FEATURE_PAL
+VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException)
```

Comment:
``DebugBreak`` introduces ``function declared 'noreturn' should not return [-Werror,-Winvalid-noreturn]`` error. Is it reasonable to use ``UNREACHABLE()`` instead of ``DebugBreak()`` here?